mmv1/third_party/terraform/tests/resource_vertex_ai_endpoint_test.go

```diff
@@ -27,6 +27,7 @@ func TestAccVertexAIEndpoint_vertexAiEndpointNetwork(t *testing.T) {
 	t.Parallel()
 
 	context := map[string]interface{}{
+		"endpoint_name": fmt.Sprint(randInt(t) % 9999999999),
```

Comment:
Is endpoint_name strictly an integer? We try to prefix all names with `tf-test-` so that sweepers can clean them up if they get left around accidentally

Reply:
Yes, endpoint name must be numeric and at most 10 digits.